src/test/java/com/suse/netapi/tests/SaltStackClientIntegrationTests.java

```diff
@@ -0,0 +1,37 @@
+package com.suse.saltstack.netapi.tests;
```

Comment:
The package is not matching the folder structure as before, please fix the folder structure (package is fine).